src/event_pusher/mod_event_pusher_push_plugin_defaults.erl

```diff
@@ -80,10 +80,10 @@ publish_notification(Acc, _, Payload, Services) ->
 -spec should_publish(Acc :: mongoose_acc:t(), To :: jid:jid()) -> boolean().
 should_publish(Acc, #jid{} = To) ->
     HostType = mongoose_acc:host_type(Acc),
-    try mongoose_users:does_user_exist(HostType, To) of
-        false ->
+    try mongoose_hooks:does_user_exist(HostType, To, stored) of
```

Comment:
Should we have a generic wrapper interface in ejabberd_auth module (e.g. `ejabberd_hook:does_user_exist/3`) that triggers the hook and returns just a boolean value? From my perspective, ejabberd_auth is the right place for such an interface.

Reply:
I like this idea very much actually.